### ISSUE TITLE
Dragging not possible if time+date is multiline

### ISSUE THREAD

**Issue by bithir** [unsoluble/smalltime#195]
If the date display spans over a line, the div for it spans over the dragging div in Chrome.
See discussion on Foundry Discord https://discord.com/channels/170995199584108546/722566083499786352/1189921715531304990

https://github.com/unsoluble/smalltime/assets/19734037/08250949-d400-4935-a08b-ede3ffdccf4a

**Comment by bithir** [comment]
I would suggest adding the following css elements for the dateDisplay div:
```css
element.style {
    text-overflow: ellipsis;
    text-wrap: nowrap;
}
```

**Comment by unsoluble** [comment]
Thanks, the sizing on all the text elements was meant to be fixed anyway, but I had a couple bits that were still using `rem`, so your larger-than-vanilla core font size was pushing the date past its intended scope.

Fixed that, and did also add an overflow control, just in case (`overflow: hidden;` is necessary too here, is why it wasn't working for you). Anyway, fixed in 1.19.2, cheers. :)
